packages/template-retail-react-app/app/constants.js

```diff
@@ -201,3 +201,35 @@ export const STORE_LOCATOR_DISTANCE = 100
 export const STORE_LOCATOR_NUM_STORES_PER_LOAD = 10
 export const STORE_LOCATOR_DISTANCE_UNIT = 'km'
 export const STORE_LOCATOR_IS_ENABLED = true
+
+// Constants for shopper context
+export const SHOPPER_CONTEXT_FIELD_TYPES = {
+    INT: 'int',
+    DOUBLE: 'double',
+    ARRAY: 'array'
+}
+export const SHOPPER_CONTEXT_SEARCH_PARAM_TO_API_FIELD_MAPPING = {
```

Comment:
Should we rename the name as well? eg `SHOPPER_CONTEXT_SEARCH_PARAMS` looks fine to me